media-proxy/include/mesh/st2110.h

```diff
@@ -0,0 +1,57 @@
+#ifndef ST2110_H
+#define ST2110_H
+
+#include <thread>
+#include <bsd/string.h>
+#include <arpa/inet.h>
+#include <mtl/st_pipeline_api.h>
+#include <mtl/st30_pipeline_api.h>
+
+#include "conn.h"
+#include "mesh_dp.h"
+#include "logger.h"
+
+namespace mesh::connection {
+
+#define ST_APP_PAYLOAD_TYPE_ST30 (111)
+#define ST_APP_PAYLOAD_TYPE_ST20 (112)
+#define ST_APP_PAYLOAD_TYPE_ST22 (114)
```

Comment:
Add inline comments referring to the SMPTE definition of that. You may include a link to documentation.

Reply:
Those numbers I took from older implementation of media_proxy.
Based on my knowledge it originate from MTL implementation
https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/app/src/fmt.h
Digging a bit deeper, it looks like ```payload_type``` can be any 7 bits number.
This field should be added into  ```MeshConfig_ST2110```

Reply:
This is not from SMPTE, If i remember correctly those are just magic numbers chosen to fit in dynamic range of payload types numbers in RTP.
https://en.wikipedia.org/wiki/RTP_payload_formats#:~:text=Payload%20identifiers%2096%E2%80%93127%20are,assigned%20port%20is%20not%20required.